app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,6 @@
+class OrdersController < ApplicationController
+  def index
+    @user = current_user
+    @order = @user.orders
```

Comment:
@jtruong2 won't @user.orders return an array of orders, even if it is only one? I wonder if we need to create an instance method on the User model for "this_order" or something that declares that "this_order" is something along the lines of.. 
```def this_order
    self.orders.last
end```.

Or is this the Current Users index of orders? If so, should `@order` be `@orders`?